app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
+  def show
+  	@user = User.find(params[:id])
```

Comment:
We probably don't need a ``show`` action, but we will need the ``edit`` and ``update`` actions for changing your  email, name, or unsubscribing.